openshift-tmp/manifests/0400_operator.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
We need the deployment as well, right? Or have I missed its existence outside of openshift-tmp?